masterblaster/masterblaster.py

```diff
@@ -42,7 +68,7 @@ async def teardown(self) -> None:
     @classmethod
     async def create(cls, access_token: str, org_name: str) -> "MasterBlaster":
         """
-        Create a new MasterBlaster instance
+        Create a new fully setup MasterBlaster instance
```

Comment:
I think it sounds better, just a small nitpick.
```suggestion
        Create a new, fully configured MasterBlaster instance
```